src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -248,6 +248,28 @@ public void disabledPeriodic() {
 			isArmCoast = !isArmCoast;
 		}
 		wasArmButtonPressed = isArmButtonPressed;
+
+		if ((subsystems.visionSubsystem != null) && (subsystems.armLedSubsystem != null)) {
+			// spotless:off
+			// Check if there is a recent target, and if the angle is
+			boolean isAlignmentCorrect =
+					subsystems.visionSubsystem.hasTargets()
+					&& ((subsystems.visionSubsystem.getLastTimestampSeconds() - Timer.getFPGATimestamp())
+						< 0.5)
+					&& (((int)(Math.toDegrees(subsystems.visionSubsystem.getRobotPose()
+																		.getRotation()
+																		.getAngle())
+						+ 361) % 360) < 2);
```

Comment:
Should this be `+ 181) % 360)`? I'm assuming the intention is to make -179 and 179 map to 2 and 0, respectively.

Reply:
Not sure - we'll figure out once this runs on the robot